### PR TITLE
Update SECURITY.md

### PR DESCRIPTION
**Pull request by scriptrunner2049**
#### Summary of Changes

Fixes #

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -15,9 +15,43 @@ Expect a response as fast as possible, within one business day at the latest.
...
+
+Consensus/Safety Violations:
+$250k USD in locked SOL tokens (locked for 12 months)
+* Consesnus safety violation
+* Tricking a validator to accept an optimistic confirmation or rooted slot without a double vote, etc..
+
+Other Attacks:
+ $100k USD in locked SOL tokens (locked for 12 months)
+* Protocol liveness attacks, 
+* Eclipse attacks,
```
Comment (mvines) [inline SECURITY.md:34]:
```suggestion
 $100k USD in locked SOL tokens (locked for 12 months)
* Protocol liveness attacks,
* Eclipse attacks,
```

File: SECURITY.md [hunk 2]
```diff
@@ -15,9 +15,43 @@ Expect a response as fast as possible, within one business day at the latest.
...
+
+Other Attacks:
+ $100k USD in locked SOL tokens (locked for 12 months)
+* Protocol liveness attacks, 
+* Eclipse attacks, 
+* Remote attacks that partition the network,
+ 
+DoS Attacks:
+ $25k USD in locked SOL tokens (locked for 12 months)
+* Non-RPC DoS,
```
Comment (mvines) [inline SECURITY.md:39]:
```suggestion
* Non-RPC DoS,
```

File: SECURITY.md [hunk 3]
```diff
@@ -15,9 +15,43 @@ Expect a response as fast as possible, within one business day at the latest.
...
+
+RPC DoS/Crashes:
+$5k USD in locked SOL tokens (locked for 12 months)
+
+
+Eligibility:
+* The participant submitting the bug bounty shall follow the process outlined within this document
+* Valid exploits can be eligible even if they are not successfully executed on the cluster
+* Multiple submissions for the same class of exploit are still eligible for compensation, though may be compensated at a lower rate, however these will be assessed on a case-by-case basis
+* Participants must complete KYC and sign the participation agreement here when the registrations are open https://solana.com/validator-registration. Security exploits will still be assessed and open for submission at all times. This needs only be done prior to distribution of tokens.
```
Comment (mvines) [inline SECURITY.md:51]:
```suggestion
* Participants must complete KYC and sign the participation agreement here when the registrations are open https://solana.com/validator-registration. Security exploits will still be assessed and open for submission at all times. This needs only be done prior to distribution of tokens.
```

File: SECURITY.md [hunk 4]
```diff
@@ -15,9 +15,43 @@ Expect a response as fast as possible, within one business day at the latest.
...
+We offer bounties for critical security issues. Please see below for more details.
+
+Loss of Funds:
+$500,000 USD in locked SOL tokens (locked for 12 months)
+* Theft of funds without users signature from any account
+* Theft of funds without users interaction in system, token, stake, vote programs
+* Theft of funds that requires users signature
+
+Consensus/Safety Violations:
+$250k USD in locked SOL tokens (locked for 12 months)
```
Comment (mvines) [inline SECURITY.md:27]:
```suggestion
$250,000 USD in locked SOL tokens (locked for 12 months)
```

File: SECURITY.md [hunk 5]
```diff
@@ -15,9 +15,43 @@ Expect a response as fast as possible, within one business day at the latest.
...
+* Consesnus safety violation
+* Tricking a validator to accept an optimistic confirmation or rooted slot without a double vote, etc..
+
+Other Attacks:
+ $100k USD in locked SOL tokens (locked for 12 months)
```
Comment (mvines) [inline SECURITY.md:32]:
```suggestion
 $100,000 USD in locked SOL tokens (locked for 12 months)
```

File: SECURITY.md [hunk 6]
```diff
@@ -15,9 +15,43 @@ Expect a response as fast as possible, within one business day at the latest.
...
+* Protocol liveness attacks, 
+* Eclipse attacks, 
+* Remote attacks that partition the network,
+ 
+DoS Attacks:
+ $25k USD in locked SOL tokens (locked for 12 months)
```
Comment (mvines) [inline SECURITY.md:38]:
```suggestion
 $25,000 USD in locked SOL tokens (locked for 12 months)
```

File: SECURITY.md [hunk 7]
```diff
@@ -15,9 +15,43 @@ Expect a response as fast as possible, within one business day at the latest.
...
+* Non-RPC DoS, 
+* Remote resource exaustion via Non-RPC protocols
+ $5k
```
Comment (mvines) [inline SECURITY.md:41]:
```suggestion
 $5,000 USD in locked SOL tokens (locked for 12 months)
```

File: SECURITY.md [hunk 8]
```diff
@@ -15,9 +15,43 @@ Expect a response as fast as possible, within one business day at the latest.
...
+
+RPC DoS/Crashes:
+$5k USD in locked SOL tokens (locked for 12 months)
```
Comment (mvines) [inline SECURITY.md:44]:
```suggestion
$5,000 USD in locked SOL tokens (locked for 12 months)
```

File: SECURITY.md [hunk 9]
```diff
@@ -15,9 +15,43 @@ Expect a response as fast as possible, within one business day at the latest.
...
+ $100,000 USD in locked SOL tokens (locked for 12 months)
+* Protocol liveness attacks,
+* Eclipse attacks,
+* Remote attacks that partition the network,
+ 
+DoS Attacks:
+ $25,000 USD in locked SOL tokens (locked for 12 months)
+* Non-RPC DoS,
+* Remote resource exaustion via Non-RPC protocols
+ $5,000 USD in locked SOL tokens (locked for 12 months)
```
Comment (mvines) [inline SECURITY.md:41]: @scriptrunner2049 - this part is a little jumbled up, are these two separate sections?

File: SECURITY.md [hunk 10]
```diff
@@ -15,9 +15,43 @@ Expect a response as fast as possible, within one business day at the latest.
...
+$250,000 USD in locked SOL tokens (locked for 12 months)
+* Consesnus safety violation
+* Tricking a validator to accept an optimistic confirmation or rooted slot without a double vote, etc..
+
+Other Attacks:
+ $100,000 USD in locked SOL tokens (locked for 12 months)
+* Protocol liveness attacks,
+* Eclipse attacks,
+* Remote attacks that partition the network,
+
```
Comment (mvines) [inline SECURITY.md:36]:
```suggestion

```

File: SECURITY.md [hunk 11]
```diff
@@ -15,9 +15,43 @@ Expect a response as fast as possible, within one business day at the latest.
...
+$5,000 USD in locked SOL tokens (locked for 12 months)
+
+
+Eligibility:
+* The participant submitting the bug bounty shall follow the process outlined within this document
+* Valid exploits can be eligible even if they are not successfully executed on the cluster
+* Multiple submissions for the same class of exploit are still eligible for compensation, though may be compensated at a lower rate, however these will be assessed on a case-by-case basis
+* Participants must complete KYC and sign the participation agreement here when the registrations are open https://solana.com/validator-registration. Security exploits will still be assessed and open for submission at all times. This needs only be done prior to distribution of tokens.
+
+Notes:
```
Comment (mvines) [inline SECURITY.md:53]:
```suggestion
Notes:
```

File: SECURITY.md (hunk 9 above)
Comment (scriptrunner2049) [reply SECURITY.md]: Good pick-up. I've edited the doc.

**Comment by mvines** [comment]
Thanks @scriptrunner2049

**Pull request merged**